plugins/psx/src/psx-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.Psx : Object, Plugin {
```

Comment:
Maybe we should name it PlayStation rather than Psx: it's longer but ore explicit.